examples/UnionExample.hs

```diff
@@ -0,0 +1,32 @@
+module UnionExample where
```

Comment:
Can you demonstrate (maybe by creating a different branch & PR that runs on CI) that a failure to compile in examples will break the build on Circle?